### PR TITLE
Drop @r1chardj0n3s, @silvester747 from list of maintainers

### PR DESCRIPTION
**Pull request by hoechenberger**
They've never accepted the auto-generated invitation, which was sent out
many times during the past couple months (with every new release).

Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hoechenberger** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -47,7 +47,5 @@ extra:
   recipe-maintainers:
     - mwcraig
     - jakirkham
-    - r1chardj0n3s
-    - silvester747
```
Comment (jakirkham) [inline recipe/meta.yaml]: @r1chardj0n3s @silvester747 , do you want to be removed as maintainers?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ source:
   sha256: 6cdfbde022999de5598d5f8e1693063ebd9efa59ff3c20d82b033bc4b39fad69
 
 build:
-  number: 0
+  number: 1
```
Comment (jakirkham) [inline recipe/meta.yaml]: No need for a build number bump.

**Comment by hoechenberger** [comment]
@r1chardj0n3s 
This is the repository that holds the build instructions ("recipe") for the `pyglet` package on [`conda-forge`](https://conda-forge.org). According to `git blame`, it appears that about 3 years ago, when the recipe was created, somebody added you to the list of maintainers – why, I cannot tell. A bot is supposed to send out invitations whenever a user is added as a maintainer; and this message is usually sent out again and again with each new change added to the recipe, until the invitee accepts the invitation. It's odd you've never received such a message!

At any rate, since you obviously didn't mean to be added as a maintainer here, we'll remove you from the list. Sorry for the inconvenience!

**Comment by r1chardj0n3s** [comment]
Thanks so much for the reply, @hoechenberger! That makes sense, I'm happy to be removed :)

**Comment by hoechenberger** [comment]
@jakirkham I think the situation might be similar with @silvester747 … If you don't mind, I'd proceed merging this PR.

**Pull request merged**